pkgs/by-name/cf/cf-hero/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "cf-hero";
+  version = "1.0.4";
+
+  src = fetchFromGitHub {
+    owner = "musana";
+    repo = "CF-Hero";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-n0kcapHBz6Dap6KKJByCwBZmXmcO/aK88X78Yit6rx4=";
+  };
+
+  vendorHash = "sha256-Yf+iZ3UIpP9EtOWW1jh3h3zTFK1D7mcOh113Q4fbAhA=";
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
+  meta = {
+    description = "Tool that uses multiple data sources to discover the origin IP addresses of Cloudflare-protected web applications";
+    homepage = "https://github.com/musana/CF-Hero";
+    changelog = "https://github.com/musana/CF-Hero/releases/tag/${finalAttrs.src.tag}";
```

https://github.com/NixOS/nixpkgs/issues/367739
```suggestion
    changelog = "https://github.com/musana/CF-Hero/releases/tag/v${finalAttrs.version}";
```

---

It could happen that we have to switch to use `rev` if we need go for unstable. But in that case the changelog URL will be broken anyways as upstream doesn't have `1.0.4-unstable-20xx-xx-xx` or alike.

Sticking to `${finalAttrs.src.tag}` reduces the maintenance from my point of view. It happens more often that upstream projects start tagging the source differently (e.g., no longer use `v`) than an override is needed when we are talking about tools. In the Python ecosystem overrides are very common.